src/utils/components/SidebarEditor/utils.ts

```diff
@@ -0,0 +1,10 @@
+import { load } from 'js-yaml';
+
+export const safeLoad = <T>(value: string): T | undefined => {
```

Comment:
what about safeDump?
if it's a generic type, I don't think nested util file under local utils is the correct place, as here it is always Resource type

Reply:
the doc about `skipInvalid` says `do not throw on invalid types (like function in the safe schema) and skip pairs and single values with such types.`. Can we trust it?

Reply:
`Resource` in SidebarEditor is a generic type. Can be VM, VMI, Template, etc.